more/free-programming-interactive-tutorials-en.md

```diff
@@ -314,6 +314,11 @@
 * [R Tutorial](https://www.w3schools.com/r) - W3Schools
 
 
+### Regex
```

Comment:
:information_source: It's usually not helpful to add a category for a single resource. :twisted_rightwards_arrows: Try to...
- ... find another category matching their topics.
- ... see if there are other books in our lists or third-party that would fit into your category (this fact contributes to consolitate it).

⚠️ Mising TOC entry

ℹ️ In other listing (books by subject / agnostic) is named `Regular Expressions`